pkg/acsengine/const.go

```diff
@@ -41,8 +41,8 @@ const (
 )
 
 const (
-	KubernetesHyperkubeImageName         = "hyperkube-amd64:v1.5.3"
-	KubernetesDashboardImageName         = "kubernetes-dashboard-amd64:v1.5.1"
+	KubernetesHyperkubeImageName         = "hyperkube-amd64:v1.6.0"
```

Comment:
For e.g. for:
-ENV KUBECTL_VERSION 1.5.3 
+ENV KUBECTL_VERSION 1.6.0 

If we have 2 consts: 1.5.3 and 1.6.0 the value can be passed to the Dockerfile and as well as the code everywhere.